tests/test_encoding/test_count_frequency_encoder.py

```diff
@@ -2,7 +2,7 @@
 import pytest
 from sklearn.exceptions import NotFittedError
 
-from feature_engine.encoding import CountFrequencyEncoder
+from feature_engine.encoding import CountFrequencyEncoder, BaseCategoricalTransformer
```

Comment:
after you remove BaseCategoricalTransformer from the init, the way to import it is:

from feature_engine.encoding.base_encoder import CountFrequencyEncoder, BaseCategoricalTransformer

Reply:
How come `CountFrequencyEncoder` and other encoders classes are included in the init file and not `BaseCategoricalTransformer`?

Also, I updated the code.

Reply:
The BaseCategoricalEncoder is an internal class, not meant to be used by Feature-engine users, thus, it is not in the init.